# Overview

Barcode sequencing technology for functional screening
Barcode-seq is a next-generation sequencing (NGS) technique that reads genome-integrated artificial sequences called barcodes that specifically mark biological materials, such as cells or genes, with unique sequences [Smith et. al., 2009]. Having a unique barcode facilitates tracking materials of interest in genome-wide functional screens as well as the identification of drug targets or disease-associated genes [Carpenter et. al., 2004]. Currently, barcode-seq is used in several genome-wide screening tools, including shRNAs for gene knock-down [Sims et. al., 2011], sgRNAs for genome editing [Parnas et. al., 2015] and barcoded yeast deletion strains in Saccharomyces cerevisiae and Schizosaccharomyces pombe for gene knock-down or knock-out [Smith et. al., 2009; Kim et. al., 2010; Eason et. al., 2004]. The use of these tools for loss-of-function studies allows the discovery of relationships between genes and a specific environment without prior knowledge. Thus, barcode-seq is an effective system for genome-wide screening studies to comprehensively understand biological systems.

# Contact manager

Questions and comments can be emailed to the system administrator of BaSDAS system (Barcode.SDAS@gmail.com).

# Citation
 
Park, Young-Kyu, et al. "BaSDAS: a web-based pooled CRISPR-Cas9 knockout screening data analysis system." Genomics & informatics 18.4 (2020).

# Data availability
 - [ ] 001 - 010: http://www.appex.kr/web_download/PSJ/BasDas/001-010.zip

 - [ ] 011 - 020: http://www.appex.kr/web_download/PSJ/BasDas/011-020.zip

※ If you get in trouble to download, mouse 'right click' and 'save as'

※ Recommended Internet Explorer or other browsers, cannot be downloaded from Google-chrome browser

# References

- Smith AM, Heisler LE, Mellor J, Kaper F, Thompson MJ, Chee M, Roth FP, Giaever G, Nislow C. Quantitative phenotyping via deep barcode sequencing. Genome Res. 2009;19(10):1836-42.

- Carpenter AE, Sabatini DM. Systematic genome-wide screens of gene function. Nat Rev Genet. 2004;5(1):11-22.

- Sims D, Mendes-Pereira AM, Frankum J, Burgess D, Cerone MA, Lombardelli C, Mitsopoulos C, Hakas J, Murugaesu N, Isacke CM, et al. High-throughput RNA interference screening using pooled shRNA libraries and next generation sequencing. Genome Biol. 2011;12(10):R104.

- Parnas O, Jovanovic M, Eisenhaure TM, Herbst RH, Dixit A, Ye CJ, Przybylski D, Platt RJ, Tirosh I, Sanjana NE, et al. A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. Cell. 2015;162(3):675-86.

- Kim DU, Hayles J, Kim D, Wood V, Park HO, Won M, Yoo HS, Duhig T, Nam M, Palmer G, et al. Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe. Nat Biotechnol. 2010;28(6):617-23.

- Eason RG, Pourmand N, Tongprasit W, Herman ZS, Anthony K, Jejelowo O, Davis RW, Stolc V. Characterization of synthetic DNA bar codes in Saccharomyces cerevisiae gene-deletion strains. Proc Natl Acad Sci U S A. 2004;101(30):11046-51.

- Li, et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biology. 2014;15:554.
